File: app/src/main/java/com/alexkn/syntact/restservice/TemplateType.java
package com.alexkn.syntact.restservice;

import com.google.gson.annotations.SerializedName;

public enum TemplateType {

    @SerializedName("Basic")
    BASIC,

    @SerializedName("Advanced")
    ADVANCED,

    @SerializedName("Custom")
    CUSTOM
}

File: app/src/main/java/com/alexkn/syntact/restservice/TemplateType.kt
package com.alexkn.syntact.restservice

import com.google.gson.annotations.SerializedName

enum class TemplateType {

    @SerializedName("Basic")
    BASIC,

    @SerializedName("Advanced")
    ADVANCED,

    @SerializedName("Custom")
    CUSTOM
}
